arangod/Cluster/ServerState.cpp

```diff
@@ -712,6 +753,22 @@ void ServerState::setId(std::string const& id) {
 /// @brief get the short server id
 ////////////////////////////////////////////////////////////////////////////////
 
+uint64_t ServerState::getRebootId() {
+  return _rebootId;
+}
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief set the short server id
+////////////////////////////////////////////////////////////////////////////////
+
+void ServerState::setRebootId(uint64_t const id) {
+  _rebootId = id;
```

Comment:
Can you assert hier that `id` is not `0`
You did outside, but if someone modifies the code later and does not move assertion it is lost.